### PR TITLE
Added Eslint Script

### PR DESCRIPTION
**Pull request by colefriedlaender**
- Creat config with `eslint-init`
- Added Nextjs typings
- Add prettier config

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
+    "eslint": "eslint",
```
Comment (lmachens) [inline package.json]:
```suggestion
    "eslint": "eslint .",
```

**Review by lmachens — CHANGES_REQUESTED** [review]
Gj, but the `eslint` script is missing a `.`. 
You can test it with `npm run eslint`.

**Comment by DNaunin** [comment]
Good Job! 🚀

**Review by ShiningElektra — APPROVED** [review]
GJ 👍

**Pull request merged**